src/app/anon/login.html

```diff
@@ -1,9 +1,20 @@
 <div class="background">
 	<div class="logInTile">
-	<form name="login.form" ng-submit="login.login()" novalidate>
-	<input type="email" name="email" ng-model="login.user.email" placeholder="Email Address" required />
-	<input type="password" name="password" ng-model="login.user.password" placeholder="Password" required />
-	<input type="submit" value="Log In" />
-	</form>
+		<p>Welcome! Please Log In!</p>
+		<ul>
```

Comment:
lets stay away from a list in this case
lets style the li's as div containers instead
look at how bootstrap does their forms: http://getbootstrap.com/css/#forms